lib/run.js

```diff
@@ -60,7 +60,7 @@ module.exports = function simpleJslint(options) {
         }
 
         if (!file.isBuffer()) {
-            var err = new gutil.PluginError(pluginName, "This plugin requires bufferred files", {});
+            var err = new gutil.PluginError(pluginName, "This plugin requires buffered files", {});
```

Comment:
Perhaps we should include the name and path of the non-buffered file in this error message? It would have been easier to diagnose https://github.com/pandell/node-repo-init/pull/148 if this information had been included in the error.